addons/mail/static/src/discuss/gif_picker/common/messaging_service_patch.js

```diff
@@ -0,0 +1,21 @@
+/* @odoo-module */
+
+import { Messaging, messagingService } from "@mail/core/common/messaging_service";
+
+import { patch } from "@web/core/utils/patch";
+
+messagingService.dependencies.push("discuss.gifPicker");
```

Comment:
Not necessary before merge, but the dependency should be in reverse, picker should depend on messaging instead.

messaging should probably store a promise that is resolved with the return of init that other services can then plug into with their own then/await